src-tauri/src/main.rs

```diff
@@ -982,6 +987,24 @@ fn main() {
                 }
             };
 
+            let token_state_clone = app.state::<UniverseAppState>().airdrop_access_token.clone();
+            let memory_state_clone = app.state::<UniverseAppState>().in_memory_config.clone();
+            app.listen("airdrop_token", move |event| {
```

Comment:
Will this continue to listen for the rest of the apps runtime? And could an airdrop token get changed at a time it's not supposed to?

Reply:
yes, and not sure about "not supposed to" but it can change if it expires (then refresh it triggered on FE) also if someone logs out and back in for whatever reason